src/ast/irbuilderbpf.cpp

```diff
@@ -407,6 +407,42 @@ Value *IRBuilderBPF::CreateStrcmp(Value* val, std::string str, bool inverse) {
   return result;
 }
 
+Value *IRBuilderBPF::CreateStrcmp(Value* val1, Value* val2, bool inverse) {
```

Comment:
Would you be able to add a comment with pseudo-C code describing what this function does, like we have for `createLog2Function` and `createLinearFunction`. It can be quite hard to see the algorithm through the LLVM IR otherwise.

Reply:
I think we'd be able to optimise it a little bit by exiting early if we detect null bytes though